.travis.yml

```diff
@@ -11,8 +11,14 @@ php:
   - 5.6
   - 7.0
 
+before_install:
```

Comment:
gosh, I'm such a pain, but if this could be moved down to be near / next to the before_script, then it would keep relevant pieces all together.

last change, I promise, then I'm happy.